OpeningTimes.js

```diff
@@ -71,17 +71,21 @@ class OpeningTimes {
     const dateTime = moment(startDateTime)
       .set({ hour: 0, minute: 0, second: 0 });
 
-    let dayCount = 0;
-    do {
-      dateTime.add(1, 'day');
-      const day = dateTime.format('dddd').toLowerCase();
-      const daysOpeningTimes = openingTimesForWeek[day];
-      if (!this.isClosedAllDay(daysOpeningTimes)) {
-        return this.getNextOpeningTimeForDay(dateTime);
-      }
-      dayCount++;
-    } while (dayCount < 7);
-    return undefined;
+    const nextOpeningTime =
+      [1, 2, 3, 4, 5, 6, 7]
+        .filter((d) => {
```

Comment:
I know there are lots of other instances where this is the case and i've not picked up/mentioned it but would be it better to use a more informative variable name other than `d`? I was thinking `dayIndex` or `dayNumber`?